.pre-commit-config.yaml

```diff
@@ -0,0 +1,40 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.6.0 # Use the ref you want to point at
+    hooks:
+      - id: trailing-whitespace
+      - id: check-added-large-files
+      - id: check-json
+      - id: check-yaml
+      - id: check-merge-conflict
+  - repo: local
+    hooks:
+      - id: backend-unit-pytest
+        name: backend unit tests with pytest
+        entry: python3 -m pytest backend
+        language: system
+        types: [python]
+        pass_filenames: false
+        files: backend
+        always_run: false
+  # Copied from https://black.readthedocs.io/en/stable/integrations/source_version_control.html
+  - repo: https://github.com/psf/black-pre-commit-mirror
+    rev: 24.4.2
+    hooks:
+      - id: black
+        language_version: python3.11
+  # TODO: uncomment once eslint configuration is fixed
```

Comment:
@denishacquin could you help me with this? I haven't been able to configure eslint properly and it's giving me some false positives

Reply:
Fixed by Denis